index.d.ts

```diff
@@ -26,6 +26,7 @@ arrify(undefined);
 declare function arrify(value: null | undefined): [];
 declare function arrify(value: string): [string];
 declare function arrify<ValueType>(value: ValueType[]): ValueType[];
+declare function arrify<ValueType>(value: ReadonlyArray<ValueType>): ReadonlyArray<ValueType>;
```

Comment:
@sindresorhus This should be the following if we would target TS 3.4:
```suggestion
declare function arrify<ValueType>(value: readonly ValueType[]): readonly ValueType[];
```

Do you want to release a new major or should we leave older syntax?

Reply:
> Do you want to release a new major or should we leave older syntax?

Leave it with a TODO comment about changing it in the next major version.

Reply:
Why would you want to change it? The new syntax does exactly the same (for arrays) and is not backwards compatible

Reply:
Just because @sindresorhus prefers new syntax.

Reply:
@sindresorhus likes his packages being in line with newest language features because of maintainability reasons, this way he simply has to make fewer releases. Could you please add the TODO comment and fix the linting issue?